bindings/2.200/GeometryDash-extras.bro

```diff
@@ -9968,7 +9968,7 @@ class LevelCell : TableViewCell {
 [[link(android)]]
 class LevelPage : cocos2d::CCLayer, DialogDelegate {
 	void onTheTower(cocos2d::CCObject* sender);
-	void onMoreGames(cocos2d::CCObject* sender);
+	void onMoreGames(cocos2d::CCObject* sender) = win 0x277a50;
```

Comment:
erm why is this the same address as menulayer::onMoreGames

Reply:
Will fix when I wake up, probably forgot to press ctrl+c 💀